velox/exec/ExchangeSource.h

```diff
@@ -139,12 +139,16 @@ class ExchangeSource : public std::enable_shared_from_this<ExchangeSource> {
 
   static std::vector<Factory>& factories();
 
+  ExchangeQueue* testingQueue() const {
+    return queue_.get();
+  }
+
  protected:
   // ID of the task producing data
   const std::string taskId_;
   // Destination number of 'this' on producer
   const int destination_;
-  const std::shared_ptr<ExchangeQueue> queue_;
+  const std::shared_ptr<ExchangeQueue> queue_{nullptr};
```

Comment:
we don't need to explicitly init it as nullptr as shared_ptr is nullptr by default and it is system independent?